addon/find-role.js

```diff
@@ -1,5 +1,9 @@
 export default function findRole(...roles) {
   const selector = roles.map((role) => `[data-role="${role}"]`).join(' ');
 
+  if()   {
```

Comment:
Expected an identifier and instead saw 'if'.<br>Expected an operator and instead saw '('.<br>Missing semicolon.